airflow/www/templates/airflow/dags.html

```diff
@@ -34,7 +34,7 @@
 {% endblock %}
 
 {% block content %}
-  <h2>DAGs</h2>
+  <h2>{{ site_title }}</h2>
```

Comment:
This one is wrong

Reply:
I think the original issue has evolved via @mik-laj's suggestion to update every page. Initially the request was to be able to amend the page title in `dags.html` so to distinguish differing installatios via the UI.

Reply:
Ahh I see